float8_experimental/fsdp_utils.py

```diff
@@ -301,7 +303,7 @@ def __tensor_flatten__(self):
             ],
             {
                 "mm_config": self._mm_config,
-                "is_amax_initialized": is_amax_initialized,
```

Comment:
`pre-commit run --all-files` complains about undefined `is_amax_initialized` in trunk. fixing it so I can commit without bypassing linter